config/locales/en/summary.yml

```diff
@@ -66,13 +66,13 @@ en:
       income_details: Income
       income_payments_details: Payments your client gets
       income_benefits_details: Benefits your client gets
-      other_income_details: 
+      other_income_details:
         one: "Other sources of income"
         other: "Other sources of income: client and partner"
       housing_payments: Housing Payments
-      outgoings_payments_details: 
-        one: Payments %{subject} makes
-        other: Payments %{subject} make
```

Comment:
This will incorrectly produce the title "Payments your partner makes" I think. Given this has a different subject (__your__ client, __their__ partner) to other titles (partner, client). I'd be inclined to overwrite the title method in the child class with something that uses the same subject as the forms. I.e.

def title
  I18n.t(
          name,
          scope: 'summary.sections',
          subject: I18n.t("dictionary.subjects.#{subject_type}")
        ) 
        end

Reply:
Also. I don't think think this text will every be plural.

Reply:
Pluralisation is to avoid:
`Payments your client and their partner **make**`

Reply:
Yes @arthurashman but in this context it will only ever be client OR partner. No?

Reply:
Nope, for outgoings the payments page is either `Client` or `Client and Partner`